projects/canopy/src/lib/carousel/carousel-item/carousel-item.component.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  HostBinding,
+  ViewEncapsulation,
+} from '@angular/core';
+
+@Component({
+  selector: 'lg-carousel-item',
+  templateUrl: './carousel-item.component.html',
+  styleUrls: ['./carousel-item.component.scss'],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LgCarouselItemComponent {
+  @HostBinding('class.lg-carousel-item') class = true;
+  @HostBinding('attr.role') role = 'tab-panel';
+
+  _selected = false;
+  @HostBinding('attr.aria-selected')
+  get selected() {
+    return this._selected;
+  }
+  set selected(selected) {
+    this._selected = selected;
+  }
```

Comment:
Sorry @dannyoz it just occurred to me that we aren't doing anything in the getter and setter here, could it just be 
```suggestion
  @HostBinding('attr.aria-selected')
  public selected = false;
```